### PR TITLE
ariq zuhdi

### PR DESCRIPTION
**Pull request by ariqzuhdi**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# *Do not* delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a live link or preview screenshot
- [ ] Images are `240 x 240` [w x h]
- [ ] Resolved merge conflicts
- [ ] Included a description of my change below

# Things done in this Pull Request

- Replace this with your change description

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -740,3 +740,9 @@
   image: danipop.jpg
   bio: Christian, supporter of traditional family(Man + Woman), coder, sportsman. In this order
   facebook: dani.avram.pop
+  
+- name: Ariq Zuhdi
+  github: ariqzuhdi
+  image: Ariq Zuhdi.jpg
+  bio: 16 years old, 9gagger, train hard to be coder
+  facebook: ariqzuhdi1
```
Comment (jigyasa-grover) [inline _data/students.yml:748]: Is your picture added in `images` folder ?

**Review by jigyasa-grover — CHANGES_REQUESTED** [review]

**Comment by ariqzuhdi** [comment]
Yes, I've added

File: _data/students.yml (hunk 1 above)
Comment (ariqzuhdi) [reply _data/students.yml:748]: yes, i've added it

**Comment by oanarosca** [comment]
Your image isn't in the `images/students` folder. Please make sure that you add it there and that the sizes are correct.

File: _data/students.yml [hunk 2]
```diff
@@ -740,3 +740,9 @@
   image: danipop.jpg
   bio: Christian, supporter of traditional family(Man + Woman), coder, sportsman. In this order
   facebook: dani.avram.pop
+  
+- name: Ariq Zuhdi
+  github: ariqzuhdi
+  image: Ariq Zuhdi.jpg
```
Comment (robbyoconnor) [inline _data/students.yml:746]: No spaces in your image name.

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
You shouldn't have spaces in your image name. Upload it to `images/students/`

**Comment by robbyoconnor** [comment]
@ariqzuhdi I have added back the template which you deleted. Go through and read it. [Read the README](https://github.com/fossasia/gci16.fossasia.org/tree/gh-pages/README.md)!

**Comment by robbyoconnor** [comment]
Do not just check the checkboxes, actually read them.

**Comment by robbyoconnor** [comment]
Hi there -- closing this. Please re-open your PULL request against `gh-pages` on your branch.

**Comment by robbyoconnor** [comment]
@Hi there @ariqzuhdi -- closing this. Please re-open your pull request against `gh-pages` on your branch.  The reason for this is so we can see that your changes work. @ariqzuhdi

**Comment by robbyoconnor** [comment]
@ariqzuhdi when you reopen your PR -- choose the first radio button.

**Pull request closed without merging**